README.md

```diff
@@ -36,4 +36,68 @@ To run all microservices docker containers, simply run the following command:
 
 To stop all containers, run the following:
 
-` docker-compose stop `
\ No newline at end of file
+` docker-compose stop `
+
+## Microservice Database Creation
+
+### Prerequisites
+* A MySQL 5.6.x server
```

Comment:
Do we want to say 5.6.x+ ?  I don't think we're doing anything unsupported in 5.7?